README.md

```diff
@@ -759,6 +768,19 @@ the following major changes to the residential modeling codebase:
 - Dropped explicit spatial lag generation in the ingest stage.
 - Lots of other bugfixes and minor improvements.
 
+### Upcoming
+
+- Infrastructure improvements
+  - Added
+    [`build-and-run-model`](https://github.com/ccao-data/model-res-avm/actions/workflows/build-and-run-model.yaml)
+    workflow to run the model using GitHub Actions and AWS Batch.
+  - Added
+    [`delete-model-run`](https://github.com/ccao-data/model-res-avm/actions/workflows/delete-model-runs.yaml)
+    workflow to delete test run artifacts in S3 using GitHub Actions.
+  - Updated [pipeline/05-finalize](pipeline/05-finalize.R) step to
+    render a performance report using Quarto and factored S3/SNS
+    operations out into \[pipeline/06-upload.R\].
```

Comment:
```suggestion
    operations out into [pipeline/06-upload.R](pipeline/06-upload.R).
```

Reply:
Done in 7458babd12d1612b6f900c001672524ef63bf7c2.